### PR TITLE
Version 1.30.0 → 1.31.0

### PR DESCRIPTION
**Pull request by Gabriel439**

### CODE REVIEW

File: dhall-json/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+1.6.3
+
+* [yaml: Single-quote date/bool string fields](https://github.com/dhall-lang/dhall-haskell/commits/master/dhall-json)
+    * If you build against `aeson-yaml` (the default) these fields will now
+      use single fields
```
Comment (sjakobi) [inline dhall-json/CHANGELOG.md]:
```suggestion
      use single quotes
```

File: dhall-json/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+1.6.3
+
+* [yaml: Single-quote date/bool string fields](https://github.com/dhall-lang/dhall-haskell/commits/master/dhall-json)
+    * If you build against `aeson-yaml` (the default) these fields will now
```
Comment (sjakobi) [inline dhall-json/CHANGELOG.md]: This sounds a bit as if `aeson-yaml` was an optional dependency for `dhall-json`.

File: dhall-json/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
+1.6.3
+
+* [yaml: Single-quote date/bool string fields](https://github.com/dhall-lang/dhall-haskell/commits/master/dhall-json)
```
Comment (sjakobi) [inline dhall-json/CHANGELOG.md]: Could you add a similar changelog entry for `dhall-yaml`?

File: dhall-json/CHANGELOG.md (hunk 2 above)
Comment (Gabriel439) [reply dhall-json/CHANGELOG.md]: Oops!  I keep forgetting that we split out `dhall-yaml` into a separate package

**Comment by Gabriel439** [comment]
@TristanCacqueray: You're welcome! 🙂

**Pull request merged**